sqlglot/expressions.py

```diff
@@ -626,6 +626,8 @@ def load(cls, obj):
     t.Type[Expression],
     t.Collection[t.Union[str, t.Type[Expression]]],
 ]
+ExpressionType = t.Union[str, Expression]
+ExpressionTypeNone = t.Union[str, Expression, None]
```

Comment:
I'd personally prefer to use `t.Optional` instead of creating another type just for `None`, but let's see what others think about this too.

Reply:
Agreed. I'll leave it as-is for now.

Reply:
the name is strange, maybe it should be StrOrExp

Reply:
i don't think it's necessary to make an alias for the optional one, just do t.Optional[StrOrExp]